server/src/http.rs

```diff
@@ -265,24 +265,33 @@ impl<Q: QueryExecutor + 'static> Service<Q> {
             .and(extract_request)
             .and(self.with_context())
             .and(self.with_proxy())
-            .and_then(|req, ctx, proxy: Arc<Proxy<Q>>| async move {
-                let result = proxy
-                    .handle_http_sql_query(&ctx, req)
-                    .await
-                    .map(convert_output);
-
-                match result {
-                    Ok(res) => Ok(reply::json(&res)),
-                    Err(e) => {
-                        if let proxy::error::Error::QueryMaybeExceedTTL { msg } = e {
-                            return Err(reject::custom(Error::QueryMaybeExceedTTL { msg }));
+            .and(self.with_read_runtime())
```

Comment:
Is this necessary?

Reply:
Maybe necessary. As I see, sql query should be executed in `read_runtime` rather than `default_runtime`.